### ISSUE TITLE
Reflection Theme: Secondary token highlight is difficult to see

### ISSUE THREAD

**Issue by yrp604** [Vector35/binaryninja-api#2506]
**Binary Ninja Version**
2861-dev

**Describe the bug**
In graph view, the contrast between a highlighted variable and a non-hilighted variable is too low, making it hard to follow dataflow.

**To Reproduce**
1. Open a binary in graph view, select a variable or register and look at other instances of that variable.

**Expected behavior**
That variable should have more pop

**Screenshots**
![image](https://user-images.githubusercontent.com/8716677/121825851-77e41200-cc69-11eb-9327-e258ead959fe.png)

**Version and Platform (required):**
 - Binary Ninja: Dev 2.4.2861
 - OS: windows
 - Version: ten
 
cc @jonpalmisc

**Comment by jonpalmisc** [comment]
Just a note as I work on this - If you look closely, you can see the last token selected was `rdx`. I personally think that `rdx` has sufficient highlighting. However, because `rdx` and `edx` are related, `edx` is also highlighted, **but not as intensely as** `rdx` since they aren't the /same/ register. This behavior is partially responsible for the low contrast observed here.
